oak-blob-plugins/src/main/java/org/apache/jackrabbit/oak/plugins/blob/datastore/DataStoreBlobStore.java

```diff
@@ -298,6 +307,12 @@ public int getMinRecordLength() {
 
     @Override
     public void close() throws DataStoreException {
+        // If marked as shared transient then delete the repository marker in close
+        if (SHARED_TRANSIENT) {
+            if (!Strings.isNullOrEmpty(getRepositoryId())) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/gh/oak-indexing/jackrabbit-oak/pullRequest?prid=5037914)